test/data/test_utils.py

```diff
@@ -45,3 +47,22 @@ def test_get_tokenizer_toktokt(self):
             data.get_tokenizer(1)
         with self.assertRaises(ValueError):
             data.get_tokenizer("some other string")
+
+    def test_text_nomalize_function(self):
+        # Test text_nomalize function in torchtext.datasets.text_classification
+        ref_lines = []
+        test_lines = []
+
+        data_path = 'test/asset/text_normalization_ag_news_test.csv'
+        with io.open(data_path, encoding="utf8") as f:
+            reader = unicode_csv_reader(f)
+            for row in reader:
+                test_lines.append(text_normalize(' , '.join(row)))
+
+        with open('../asset/text_normalization_ag_news_ref_results.test') as ref_data:
```

Comment:
Please use io.open here as well, because the reference is presumably encoded in utf-8